pretext/ExtraTopics/Pseudocode.ptx

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-pseudocode">
+  <title>Pseudocode</title>
+
+  <p><term>Pseudocode</term> is a mix of English and code (in our case, Python) where 
+    programmers can plan out their programs.</p>
+
+  <p>Writing pseudocode can help if you're wanting to plan before you write
+    a program, take on a large or confusing programming task, or just as a way to
+    document your code with comments to make it more readable for yourself and others.</p>
```

Comment:
I would rewrite this in present tense. 
Currently it's in imperfect tense or what ever the one that is dependent on an event occurring.